doc/source/user_guide/ug_case_settings.rst

```diff
@@ -365,7 +369,8 @@ domain_nml
    "", "``spiralcenter``", "distribute blocks via roundrobin from center of grid outward in a spiral", ""
    "", "``wghtfile``", "distribute blocks based on weights specified in ``distribution_wght_file``", ""
    "``distribution_wght``", "``block``", "full block weight method with land block elimination", "``latitude``"
-   "", "``blockall``", "full block weight method without land block elimination", ""
+   "", "``blockall``", "block weight method with NO land block elimination", ""
+   "", "``blockfull``", "full block weight method with NO land block elimination", ""
```

Comment:
I don't understand the difference.  Maybe for blockfull:  block weight method including minimal all-land block weights

Reply:
I have updated the distribution_wght documentation